src/core/sys/posix/mqueue.d

```diff
@@ -0,0 +1,218 @@
+/**
+ * D header file for Posix Message Queues
+ *
+ * Defines external functions required to manage Posix Message Queues
+ *
+ * mq_open(3)          open a message queue
+ * mq_close(3)         close a message queue
+ * mq_unlink(3)        remove a message queue
+ * mq_send(3)          send a message
+ * mq_receive(3)       receive a message
+ * mq_timedsend(3)     send a message with a timeout (linux specific)
+ * mq_timedreceive(3)  receive a message with a timeout (linux specific)
+ * mq_getattr(3)       get message queue attributes
+ * mq_setattr(3)       set message queue attributes
+ * mq_notify(3)        register asynchronous notify
+ *
+ * Copyright: Copyright (c) 2016 sociomantic labs. All rights reserved
+ * License:   $(WEB www.boost.org/LICENSE_1_0.txt, Boost License 1.0).
+ * Authors:   Andreas Bok Andersen, Mathias Lang
+ * Standards: POSIX.1-2001.
+ * See_Also:  $(WEB pubs.opengroup.org/onlinepubs/9699919799/basedefs/mqueue.h.html, Standard)
```

Comment:
Why ?
I took a look at the other `core.sys.posix` modules, and they all have the same (different because less documented) headers, but no copyright in the description.